scripts/mutations/mutations.html

```diff
@@ -0,0 +1,101 @@
+<html>
+  <head>
+    <title>Distance comparison</title>
+      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" />
+<style>
+body {
+  font-family: 'Roboto', sans-serif;
+}
+.model {
+  width: 150px;
+  height: 150px;
+  min-width: 150px;
+  background-position: 60% 50%;
+  background-size: 300px;
+}
+table {
+  padding: 0;
+  border: 0;
+  border-collapse: collapse;
+}
+th {
+  width: 100px;
+  font-weight: normal;
+}
+th, td {
+  text-align: center;
+  padding: 10px;
+}
+</style>
+  </head>
+  <body>
+<script type='text/javascript'>
+const raw = `0,60,15
+1,55,6
+2,50,7
+3,45,10
+4,40,9
+5,35,5
+6,30,3
+7,25,6
+8,20,6
+9,15,4
+10,10,4
+11,5,3
+12,0,2`;
+
+const data = raw.split('\n').map(line => line.split(',').map(n => parseInt(n)));
```

Comment:
nit: If we're hardcoding this raw data, why not just make this array off the bat?